javascript/javascript3/week2/fetchigAndWaiting.js

```diff
@@ -0,0 +1,27 @@
+setTimeout(() => {
```

Comment:
There is some overlap and duplication of functionality here. Did you mean to do same in both.
I don't understand the name fetchingAfterWating. I don't see you waiting for anything.
If you meant to demonstrate fetching using .then() and async/await then they are both ok and do the same

Reply:
Ah ok, you are waiting after you fetch. Got it